pkgs/applications/networking/cluster/helm/plugins/helm-dt.nix

```diff
@@ -0,0 +1,47 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+  ...
+}:
+buildGoModule rec {
+  pname = "helm-dt";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "vmware-labs";
+    repo = "distribution-tooling-for-helm";
+    rev = "v${version}";
+    sha256 = "sha256-KrQAlB0ORNzKIG2vxych3gVBytTh3Hhnjsyn1ia1ZQM=";
+  };
+
+  vendorHash = "sha256-T8Kk+9NAhYOvSq94HOEE53BT7Xh9tU1gJ420o/tiVEo=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X 'main.BuildDate=1970-01-01 00:00:00 UTC'"
+    "-X 'main.Commit=v${version}'"
+  ];
+
+  # NOTE: Remove the install and upgrade hooks.
+  postPatch = ''
+    sed -i '/^hooks:/,+2 d' plugin.yaml
```

Comment:
you may want to try using `substituteInPlace` if possible
```nix
# dont forget to import it too!
substituteInPlace plugin.yaml \
  --replace-fail "what you want to replace" "new content"
```

Reply:
I do not think I can replace with `substituteInPlace` because it seems that it is a one for one replace, but this `sed` is removing the `hooks` scripts section from the `plugin.yaml` file.

[`link`](https://github.com/vmware-labs/distribution-tooling-for-helm/blob/main/plugin.yaml#L6-L8)